File: 2026-merge-triplets-to-form-target-triplet/merge-triplets-to-form-target-triplet.py
class Solution:
    def mergeTriplets(self, triplets: List[List[int]], target: List[int]) -> bool:
        result = set()
        for t in triplets:
            if t[0] > target[0] or t[1] > target[1] or t[2] > target[2]:
                continue
            
            for i, v in enumerate(t):
                if v == target[i]:
                    result.add(i)
        return len(result) == 3

File: 2026-merge-triplets-to-form-target-triplet/merge-triplets-to-form-target-triplet.cpp
class Solution {
public:
    bool mergeTriplets(vector<vector<int>>& triplets, vector<int>& target) {
        unordered_set<int> result;
        for (vector<int> temp: triplets){
            if (temp[0] > target[0] || temp[1] > target[1] || temp[2] > target[2]) continue;
            for (int i = 0; i < 3; i++)
                if (temp[i] == target[i]) result.insert(i);
        }
        return result.size() == 3;
    }
};